fire/api/firedb.py

```diff
@@ -54,10 +54,17 @@ def __init__(self, connectionstring=None, debug=False):
         else:
             self.connectionstring = self._build_connection_string()
 
+        # Kører vi under test setup?
+        if self.connectionstring.startswith("fire.fire@localhost"):
+            exe_opt = {}
+        else:
+            exe_opt = {"schema_translate_map": {None: "fire_adm"}}
```

Comment:
Det er fint nok for nu, men på et tidspunkt har vi både en produktionsdatabase og en testdatabase (ej at forveksle med database til brug i testsuiten), og så er dette ikke tilstrækkeligt. Jeg tror den smarteste løsning er at schemaet være konfigurerbart i `fire.ini`.

Reply:
Det er bestemt også meningen - men det kræver lidt mere omfattende restrukturering af den omliggende kode (vi skal læse ini-filen der hvor vi sætter session op, for vi skal bruge yderligere info. Alternativt skal der returneres en dict med al relevant info fra den valgte connection - som i øvrigt skal kunne vælges fra kommandolinjen.

Hvis jeg forstår DBA-drengene korrekt så vil det nuværende dog også virke når vi kører separat testdatabase: Primært skema vil stadig være fire_adm i den.